packages/@sanity/vision/tsconfig.json

```diff
@@ -8,5 +8,9 @@
 
     "jsx": "react-jsx"
   },
-  "references": [{"path": "../types/tsconfig.lib.json"}, {"path": "../../sanity/tsconfig.lib.json"}]
+  "references": [
+    {"path": "../schema/tsconfig.json"},
+    {"path": "../types/tsconfig.json"},
+    {"path": "../../sanity/tsconfig.json"}
```

Comment:
I changed this from importing *lib* to not, I think this is correct?

Reply:
🤔 I think that's right, yes.